tests/pkg/utils/client.go

```diff
@@ -31,7 +31,7 @@ func GetKubeClientDynamic(opt TestOptions, isHub bool) dynamic.Interface {
 	if !isHub && len(opt.ManagedClusters) > 0 {
 		url = opt.ManagedClusters[0].ClusterServerURL
 		kubeConfig = opt.ManagedClusters[0].KubeConfig
-		kubeContext = ""
+		kubeContext = opt.ManagedClusters[0].KubeContext
```

Comment:
It should be addressed in your previous PR. correct?

Reply:
yes, let me rebase it, thanks.